File: Cetvrta/cetvrta.py
n = list(input().split())
l1 = [int(n[0]), int(n[1])]

n = list(input().split())
l2 = [int(n[0]), int(n[1])]

n = list(input().split())
l3 = [int(n[0]), int(n[1])]

n.clear()
if l1[0] == l2[0]:
	n.append(l3[0])
elif l1[0] == l3[0]:
	n.append(l2[0])
elif l2[0] == l3[0]:
	n.append(l1[0])

if l1[1] == l2[1]:
	n.append(l3[1])
elif l1[1] == l3[1]:
	n.append(l2[1])
elif l2[1] == l3[1]:
	n.append(l1[1])

print(n[0], n[1])

File: Cetvrta/cetvrta.rb
a, b = gets.chomp.split.map(&:to_i)
l1 = [a, b]

a, b = gets.chomp.split.map(&:to_i)
l2 = [a, b]

a, b = gets.chomp.split.map(&:to_i)
l3 = [a, b]

l4 = []
if l1[0] == l2[0]
	l4.push(l3[0])
elsif l1[0] == l3[0]
	l4.push(l2[0])
elsif l2[0] == l3[0]
	l4.push(l1[0])
end

if l1[1] == l2[1]
	l4.push(l3[1])
elsif l1[1] == l3[1]
	l4.push(l2[1])
elsif l2[1] == l3[1]
	l4.push(l1[1])
end

print("#{l4[0]} #{l4[1]}")
